types/modules/consumer.d.ts

```diff
@@ -0,0 +1,36 @@
+import { ChannelConfig } from './channels';
+import { Connection } from './connection';
+import { ConsumerHooks } from './hooks';
+import type amqp = require('amqplib');
+
+declare class Consumer {
+  constructor(connection: Connection);
+  hooks: ConsumerHooks;
+  set connection(value: Connection);
+  get connection(): Connection;
+  /**
+   * Sends the RPC reply to the response queue according to the message properties when required.
+   * @param  messageProperties   An amqp.node message properties object, containing the rpc settings
+   * @param  queue The initial queue on which the handler received the message
+   * @param  reply the received message to reply the rpc if needed:
+   * @return The message properties if it is not an rpc request, or a boolean indicating the produce result when an rpc response was produced.
+   */
+  checkRpc(
```

Comment:
I think this method should be private and not public

Reply:
What do you mean by "documented as if it is public" ? all methods are documented in this project - both public & private.